client/web/src/hooks/node-data.ts

```diff
@@ -142,20 +182,32 @@ export default function useNodeData() {
   const routesPOST = useCallback(
     (d: RoutesPOSTData) => {
       setIsPosting(true)
-      const onComplete = () => {
+      const onComplete = (success: boolean) => {
```

Comment:
Instead of passing `success` through `onComplete`, maybe just write the metric after line 201, from the `then` of the fetch.